ParameterTrendPercentages/ParameterTrendPercentages_1/ParameterTrendPercentages_1.cs

```diff
@@ -147,19 +149,8 @@ public OnPrepareFetchOutputArgs OnPrepareFetch(OnPrepareFetchInputArgs args)
         if (_parameter == null)
```

Comment:
This check here could have been avoided if you throw during the processing of the arguments (line 96), which is the best place to validate arguments.